packages/blade/src/components/Skeleton/_decisions/decisions.md

```diff
@@ -0,0 +1,327 @@
+
+  return (
+    <>
+      {isLoading ? (
+        <Box>
+          <Box display="flex">
+            <Box>
+              <Skeleton width="40%" height="30px" />
+              <Skeleton width="30%" height="20px" />
+            </Box>
+            <Skeleton width="30px" height="50px" />
+          </Box>
+          <Skeleton width="100%" height="100px" />
+        </Box>
+      ) : (
+        <Card>
+          <CardHeader>
+            <CardHeaderLeading title={data.title} subtitle={data.subtitle} />
+            <CardHeaderTrailing
+              visual={<CardHeaderBadge variant="neutral">{data.badge}</CardHeaderBadge>}
+            />
+          </CardHeader>
+          <CardBody>
+            <Text>{data.content}</Text>
+          </CardBody>
+        </Card>
+      )}
+    </>
+  );
+};
+```
+
+If only body of the card needs to be lodable consumers can put the skeleton on the CardBody:
+
+```jsx
+const App = () => {
+  const { data, isLoading } = useQuery();
+
+  return (
+    <Card>
+      <CardHeader>
+        <CardHeaderLeading title="Payment options" subtitle="Secured by Razorpay" />
+        <CardHeaderTrailing visual={<CardHeaderBadge variant="neutral">PENDING</CardHeaderBadge>} />
+      </CardHeader>
+      <CardBody>
+        {isLoading ? <Skeleton width="100%" height="100px" /> : <Text>{data.content}</Text>}
+      </CardBody>
+    </Card>
+  );
+};
+
+```
+
+A question that might arise is "Why not just give isLoading prop in the Card?"
+
+Like: 
+
+```jsx
+<Card isLoading={isLoading}>
+  <CardHeader>
+    <CardHeaderLeading title={data.title} subtitle={data.subtitle} />
+    <CardHeaderTrailing visual={<CardHeaderBadge variant="neutral">{data.badge}</CardHeaderBadge>} />
+  </CardHeader>
+  <CardBody>
+    <Text>{data.content}</Text>
+  </CardBody>
+</Card>
+```
+
+We discussed this, while it might look simple this approach also has few downsides: 
+
+- if we provide `isLoading` prop it might cause a lot of jumps and shifts on the page because we won't know until API responds if the CardHeader has title/prefix or subtitle set or not and it could be removed after the API is done loading.
+- Providing `isLoading` prop for all Blade components (where applicable) might not be the most flexible approach.
+- With `isLoading` prop consumers will anyways have to opt out and use custom Skeleton templates for the CardBody content so there's not much benefit to providing it out of the box.
+
+
+## Motion
+
+Check the motion [here](https://www.figma.com/proto/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=16502-258345&scaling=min-zoom&page-id=16498%3A256331):
+
+https://github.com/razorpay/blade/assets/35374649/8298efde-f977-4aa3-bc2d-92fb50a8c830
+
+
+## Accessibility
+
+- All Skeleton nodes will have aria-hidden to hide them from screen readers
+- All Skeleton nodes will have `cursor: progress`
+- Consumers will have to add aria-busy in the respective container which is being rendered
```

Comment:
add a code snippet for this?

Reply:
Don't have a solid working example right now, I will add accessibility story in the storybook itself. There are few nitty gritties i need to handle and test for first. 

While the reference blog that I linked does suggest to use aria-busy, i need to see can the loading state be communicated via our `LiveAnnouncer` component also?